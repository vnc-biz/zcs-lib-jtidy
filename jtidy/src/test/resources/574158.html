<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
<title>[574158] Error with FONT tag</title>
</head>
<body>
<FONT SIZE="4"><DIV ID="Etiquette"><A
HREF="http://indexation.inria.fr:8080/cgi-bin/query?mss=RRRT%2Fsimple&amp;pg=q&amp;what=web&amp;enc=iso88591&amp;q=dc.creator%3DOniboni%2CJulien+url%3Ahttp%3A%2F%2Fwww.inria.fr%2Frrrt%2F&amp;q1=dc.creator%3DOniboni%2CJulien&amp;searchrrrt=Chercher"ONMOUSEOVER="MM_showHideLayers('document.layers[\'Etiquette2\']','document.all[\'Etiquette2\']','show','document.layers[\'AuteurFenetre\']','document.all[\'AuteurFenetre\']','show')"
ONMOUSEOUT="MM_showHideLayers('document.layers[\'AuteurFenetre\']','document.all[\'AuteurFenetre\']','hide','document.layers[\'Etiquette2\']','document.all[\'Etiquette2\']','hide')"><!--div
class=aut-->Oniboni, Julien<!--/div--></A>
</FONT></DIV>
</body>
</html>
