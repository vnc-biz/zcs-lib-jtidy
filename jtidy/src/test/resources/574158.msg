<?xml version="1.0" encoding="ISO-8859-1"?>
<!-- expected messages for test 574158-->
<messages>
  <message>
    <code>-1</code>
    <level>0</level>
    <line>0</line>
    <column>0</column>
    <text><![CDATA[
Tidy (vers 4th August 2004) Parsing "InputStream"
]]></text>
  </message>
  <message>
    <code>7</code>
    <level>2</level>
    <line>7</line>
    <column>16</column>
    <text><![CDATA[missing </font> before <div>]]></text>
  </message>
  <message>
    <code>23</code>
    <level>2</level>
    <line>7</line>
    <column>16</column>
    <text><![CDATA[trimming empty <font>]]></text>
  </message>
  <message>
    <code>15</code>
    <level>2</level>
    <line>7</line>
    <column>36</column>
    <text><![CDATA[inserting implicit <font>]]></text>
  </message>
  <message>
    <code>110</code>
    <level>0</level>
    <line>1</line>
    <column>1</column>
    <text><![CDATA[InputStream: Doctype given is "-//W3C//DTD XHTML 1.0 Strict//EN"]]></text>
  </message>
  <message>
    <code>111</code>
    <level>0</level>
    <line>1</line>
    <column>1</column>
    <text><![CDATA[InputStream: Document content looks like XHTML 1.0 Transitional]]></text>
  </message>
  <message>
    <code>-1</code>
    <level>0</level>
    <line>0</line>
    <column>0</column>
    <text><![CDATA[3 warnings, no errors were found!
]]></text>
  </message>
  <message>
    <code>8</code>
    <level>0</level>
    <line>1</line>
    <column>1</column>
    <text><![CDATA[You are recommended to use CSS to specify the font and
properties such as its size and color. This will reduce
the size of HTML files and make them easier to maintain
compared with using <FONT> elements.
]]></text>
  </message>
</messages>
