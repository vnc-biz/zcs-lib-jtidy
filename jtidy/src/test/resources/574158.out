<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
<title>[574158] Error with FONT tag</title>
</head>
<body>
<div id="Etiquette"><font size="4"><a href="http://indexation.inria.fr:8080/cgi-bin/query?mss=RRRT%2Fsimple&amp;pg=q&amp;what=web&amp;enc=is
o88591&amp;q=dc.creator%3DOniboni%2CJulien+url%3Ahttp%3A%2F%2Fwww.inria.fr%2Frrrt%2F&amp;q1=dc.creator%3DOniboni%2CJulien&amp;searchrrrt=Che
rcher" onmouseover="MM_showHideLayers('document.layers[\'Etiquette2\']','document.all[\'Etiquette2\']','show','document.layers[\'AuteurFenet
re\']','document.all[\'AuteurFenetre\']','show')" onmouseout="MM_showHideLayers('document.layers[\'AuteurFenetre\']','document.all[\'AuteurF
enetre\']','hide','document.layers[\'Etiquette2\']','document.all[\'Etiquette2\']','hide')"><!--div
class=aut-->Oniboni, Julien<!--/div--></a></font></div>
</body>
</html>
